montecarlo/cfast_mcarlo.py

```diff
@@ -254,7 +260,7 @@ def _section_vvent(self):# {{{
             vent_to_name = v['vent_to_name'].split(".")[0]
```

Comment:
splitting . is already done here

Reply:
fixed in https://github.com/aamks/aamks/pull/153/commits/a17293b11a592ef6bfbf60972eec5c4aa479cae6